.changelog/2848.breaking.md

```diff
@@ -0,0 +1,4 @@
+`oasis-net-runner`: `--net.*` flags renamed to `--fixture.default.*`
```

Comment:
This will probably require updates in runtime and elsewhere. cc @peterjgilbert

Reply:
I found scripts/gateway.sh using `--net.*` flags and updated those. PR: https://github.com/oasislabs/oasis-runtime/pull/993